README.md

```diff
@@ -1109,12 +1102,11 @@ the WebKit rendering engine.
 - [Termius](https://www.termius.com/) - Cross-platform terminal with built-in SSH and Telnet.
 - [![Open-Source Software][OSS Icon]](https://github.com/lanoxx/tilda) [Tilda](https://github.com/lanoxx/tilda) - A Gtk based drop down terminal for Linux and Unix.
 - [![Open-Source Software][OSS Icon]](https://github.com/gnunn1/tilix) [Tilix](https://gnunn1.github.io/tilix-web/) - A tiling terminal emulator for Linux using GTK+ 3.
-- [![Open-Source Software][OSS Icon]](https://github.com/tmux/tmux) [Tmux](https://github.com/tmux/tmux) - It lets you switch easily between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal. And do a lot more.
```

Comment:
I guess removing tmux was a typo ?!

btw, the github org have the link http://tmux.github.io/ which redirects to the github wiki page https://github.com/tmux/tmux/wiki

Reply:
Duplicate entry, tmux is also in the Tools sub-category.

Reply:
ok